### DISCUSSION TITLE
16 series graphics cards output error and can not output larger resolution problem

### DISCUSSION THREAD

**Discussion by baiwei666** [hlky/stable-diffusion-webui#627] [Q&A]
The output of a 16 series graphics card is a green picture, how to solve it? And 6g video memory cannot output pictures with a resolution of more than 512*512. How to set it to break through this limit?

**Comment by ack226** [comment]
You'll need to use the following options:

`--precision full --no-half --optimized`

I have a GTX 1660 Super and I was able to use `--optimized-turbo` instead of `optimized`, which works fine and made it run much faster, so you should try that and if it crashes revert back to the slower `--optimized` option.

I also added `--extra-models-cpu` to force the upscaling models to use CPU since my GPU doesn't have enough memory.

My full arguments line for the GTX 1660 Super (6GB) is:

`--precision full --no-half --optimized-turbo --extra-models-cpu`

With the above everything seems to work great!

**Reply by baiwei666** [reply]
`python scripts/webui.py --precision full --no-half --optimized-turbo --extra-models-cpu`
I've added that code and it outputs a normal image, but still can't generate a higher resolution image. Program prompts me “CUDA out of memory”

**Reply by ack226** [reply]
As far as I know, 512x512 is about as high as you'll be able to go with 6GB of video memory. Most people, even with better cards, generate 512x512 images and then upscale them afterwards.

**Reply by ignotusandri** [reply]
Hey :) I'm having similiar problems.
I'm running stable diffusion on a Laptop with a GTX1650 Ti with 4GiB of vram. When using optimized or optimized turbo I am able to run it, however I only get a green square image as a result. Following the advice here I've added the suggested code. However now when I het the generate button I immediately get a crash message due to a lack of CPU. 
![grafik](https://user-images.githubusercontent.com/113249741/189488828-30b56f1f-63e5-41a1-94a6-513600a7e96c.png)

**Reply by mchaker** [reply]
4GB may be too small to run at full precision
